src/api/course/openApi.json

```diff
@@ -378,6 +378,19 @@
         "tags": ["course"],
         "summary": "Cancel / clear course.",
         "description": "Clear all course information.",
+        "parameters": [
+          {
+            "name": "force",
+            "in": "query",
+            "description": "Force HTTP only mode, ignoring all subsequent stream input. To clear this mode, call this endpoint without specifying `force`.",
+            "example": "/course/force=true",
```

Comment:
I thought I wrote about this in my review comments, but can't find it anywhere...

Anyway, I think `force` is too generic and does not really convey that this is actually causing a side effect, changing a permanent setting.  How about `setCommandSource=true` or something along those lines?

And now that commandSource is in the http API is this the best method, or should there be an explicit endpoint for resetting commandSource?

PS1: The example is not using a query parameter.

PS2: I find the expression "stream input" a bit ambiguous. How about _ignoring subsequent destination value delta updates_?

Reply:
updated force to `apiMode`
Added DELETE request to `commandSource` endpoint to clear it.

Reply:
Done